clients/drcachesim/drcachesim.dox.in

```diff
@@ -721,6 +721,14 @@ running with "-data_prefetcher none" (see \ref sec_drcachesim_ops).
 While misses from software prefetches are included in cache miss files,
 misses from hardware prefetches are not.
 
+The cache simulator can be used to analyze the stream of L2 cache miss
```

Comment:
How about making this a new \section so it shows up in the table of contents?  I think that would be useful, so someone scanning the contents will see this feature.

Reply:
Done.